spec/test_runner_spec.rb

```diff
@@ -7,6 +7,22 @@
   let(:file_multi) { File.new('spec/data/sample_multi.rb') }
   let(:file_failed) { File.new('spec/data/sample_failed.rb') }
 
+  describe '#compile_file_content' do
+    let(:test) { %q{
+y = 25
+#...content...#
+expect(x).to eq 35 } }
+    let(:expected) { %q{
+x = y + 10
+
+y = 25
+x = y + 10
```

Comment:
Hey, no, wait! It is working. The problem is that this a unit test for the test runner component, but the interpolation is done very early in the pipeline. Let's look at the following integration test - that actually checks this is working -: 

```ruby
it 'supports tests with interpolations' do
    response = bridge.run_tests! content: 'a = b',
                                test: "
                                describe do
                                  it do
                                    b = 6
                                    #...content...#
                                    expect(a).to eq 6
                                  end
                                end",
                                expectations: []

    expect(response).to eq expectation_results: [],
                           feedback: '',
                           response_type: :structured,
                           result: '',
                           status: :passed,
                           test_results: [{:title=>"should eq 6", :status=>:passed, :result=>''}]
  end
```